language/sample-apps/chat-streamlit/app.py

```diff
@@ -0,0 +1,75 @@
+import streamlit as st
+from PIL import Image
+from src.utils import *
+from src.vertex import *
+
+st.set_page_config(
+    page_title="Vertex PaLM Text Generation API",
+    page_icon=":robot:",
+    layout="centered",
+    initial_sidebar_state="expanded",
+    menu_items={
+        'Get Help': 'https://www.extremelycoolapp.com/help',
```

Comment:
This URL actually exists --> could you change it to something else that definitely does not exist?

Reply:
I have remove the Get Help and Report a bug items. The About item can remain.